src/main/java/net/pms/database/MediaTableFailedLookups.java

```diff
@@ -32,6 +32,16 @@
 public final class MediaTableFailedLookups extends MediaTable {
 	private static final Logger LOGGER = LoggerFactory.getLogger(MediaTableFailedLookups.class);
 	public static final String TABLE_NAME = "FAILED_LOOKUPS";
+	private static final String COL_LASTATTEMPT = "LASTATTEMPT";
+	public static final String LASTATTEMPT = TABLE_NAME + "." + COL_LASTATTEMPT;
+	public static final String FAILUREDETAILS = TABLE_NAME + ".FAILUREDETAILS";
+	public static final String FILENAME = TABLE_NAME + ".FILENAME";
```

Comment:
I think it could be confusing having this variable (and others) named just `FILENAME`. Maybe there is a naming convention that gives context, like `TABLE_COL_FILENAME`